### DISCUSSION TITLE
Quasar Framework Suggestion

### DISCUSSION THREAD

**Discussion by rafael-neri** [JenuelDev/Believers-Sword#81] [Ideas]
The Quasar Framework was built using Vu3 + Material Design.

It is possible to build with the same tool:
- A SPA website with PWA features (With Vue3)
- A mobile app (With Capacitor)
- A desktop application (With Electron)
- Use Server Side features (Express or other)

**Comment by JenuelDev** [comment]
I was also thinking about using Quasar,, I actually started planning on how I can move to Quasar.. especially the I have a lot of moving parts. Like Sqlite Management for offline use, But Ill try to do so.. I also like Quasar.

if it's impossible to move,  I have to find another way to do so.. :)

**Reply by rafael-neri** [reply]
I can help. I'll create a fork for this.
